include/rangers-api/Application/UI/UIMusicSelect.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+namespace app::ui {
+    class UIMusicSelect : public hh::game::GameObject, public hh::ui::UIListener, public hh::hid::ActiveDeviceManagerListener, public app::snd::CustomSoundListener {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.